src/evidently/features/is_valid_sql_feature.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+from typing import ClassVar
+from typing import Optional
+
+import sqlvalidator
+
+from evidently import ColumnType
+from evidently.features.generated_features import ApplyColumnGeneratedFeature
+
+
+class IsValidSQL(ApplyColumnGeneratedFeature):
+    class Config:
+        type_alias = "evidently:feature:IsValidSQL"
+
+    __feature_type__: ClassVar = ColumnType.Categorical
+    display_name_template: ClassVar = "SQL Validity Check for {column_name}"
+
+    column_name: str
+
+    def __init__(self, column_name: str, display_name: Optional[str] = None):
+        self.column_name = column_name
+        self.display_name = display_name
+        super().__init__()
+
+    def apply(self, value: Any):
+        if value is None or not isinstance(value, str):
+            return False
+
+        return self.is_valid_sql(value)
+
+    def is_valid_sql(self, query: str) -> bool:
+        queries = query.strip().split(";")  # Split by semicolon
```

Comment:
Let's consider the following sql

```sql
SELECT * FROM your_table WHERE your_column LIKE '%;%';
```

or

```sql
SELECT CONCAT('Hello', '; ', 'World') AS greeting_with_semicolon;
```

we will get invalid in your implementation but it is actually valid sql

Do we actually need to split by `;`?

Reply:
The split by `;` is a bit of a stretch since the column could contain multiple SQL queries. However, we can stick to the basic usage and treat the column values as single queries. 
How do we view the scope of this feature?